GoogleForCreators/web-stories-wp
# Story Data: Single Animation Target

## Summary
- Used node script to update exisiting template data animation data to be single target
- Updated story-anim-tool to apply a separate animation to each selected target
- Added migration script to make any existing stories created from outdated templates to only have a single target per animation

## Relevant Technical Choices
used node script to update existing template data, then realized this step should probably be part of the migration step cus ppl may have stories created from the old template data.

## To-do
Seems there was a regression in the story anim tool where the story animation data updates no longer hit the story api. It appears `updateStory` was altered recently in the dashboard to only be able to update the story title.

This regression feels a little separate from this ticket, I'll create a separate PR to fix it. Not sure how much we'll be using the story-anim-tool from now on to actually edit story animations anyways 🤷 

## User-facing changes
None

## Testing Instructions
Enable animations and see that they still play & work in all the templates and on stories with animations applied.

---



Fixes #4153
